### PR TITLE
EN-18249: Improvements for geocoding-secondary

### PR DESCRIPTION
**Pull request by aerust**

### CODE REVIEW

File: src/main/scala/com/socrata/geocodingsecondary/RegionCodingHandler.scala [hunk 1]
```diff
@@ -63,6 +65,8 @@ abstract class AbstractRegionCodingHandler(http: HttpClient,
                                            retries: Int) extends ComputationHandler[SoQLType, SoQLValue] {
   val log = LoggerFactory.getLogger(classOf[AbstractRegionCodingHandler])
 
+  val taskSupport = MDCTaskSupport(new ForkJoinTaskSupport(new scala.concurrent.forkjoin.ForkJoinPool(2))) _
```
Comment (chitang) [inline src/main/scala/com/socrata/geocodingsecondary/RegionCodingHandler.scala]: Is this stray code?

File: build.sbt [hunk 2]
```diff
@@ -11,7 +11,11 @@ val computationStrategies   = "com.socrata" %% "computation-strategies"     % "0
 
 val geocoders               = "com.socrata" %% "geocoders"                  % "2.0.7"
 
-val dataCoordinator         = "com.socrata" %% "secondarylib-feedback"      % "3.3.16"
+val dataCoordinator         = "com.socrata" %% "secondarylib-feedback"      % "3.3.19"
+
+val javaxServlet            = "javax.servlet" % "javax.servlet-api"         % "3.1.0" // needed for socrata-http-server
```
Comment (chitang) [inline build.sbt]: Need to add javaxServlet and socrataHttpServer just for RequestId?  Should probably do it some other way like factoring out RequestId?

File: src/main/scala/com/socrata/geocodingsecondary/RegionCodingHandler.scala (hunk 1 above)
Comment (aerust) [reply src/main/scala/com/socrata/geocodingsecondary/RegionCodingHandler.scala]: oh oops... yeah

File: build.sbt (hunk 2 above)
Comment (aerust) [reply build.sbt]: I agree that it feels like a lot... but I really want to use RequestId / don't really want to spend more time on details like that.

Comment (chitang) [reply build.sbt]: OK.  let's comment that in the dependency that we are just using RequestId.

**Comment by chitang** [comment]
:shipit:

**Pull request merged**